frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/review-tab.scss

```diff
@@ -5,3 +5,56 @@
 .kubevirt-create-vm-modal__review-tab-lower-section {
   margin-top: 20px;
 }
+
+.kubevirt-create-vm-modal__review-tab-section-container {
+  margin-top: var(--pf-global--spacer--md);
+}
+
+.kubevirt-create-vm-modal__review-tab-section {
+  padding-bottom: var(--pf-global--spacer--xl);
+  &__title {
+    margin-top: var(--pf-global--spacer--lg);
+    margin-bottom: var(--pf-global--spacer--md);
+  }
+  &__content {
+    // padding-left: var(--pf-global--spacer--lg);
+
+    &.pf-c-content ol {
+      --pf-c-content--ol--MarginLeft: 0;
+      --pf-c-content--ol--PaddingLeft: var(--pf-global--spacer--md);
+    }
+  }
+}
+
+.kubevirt-create-vm-modal__review-tab__data-list {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  @media screen and (min-width: 576px) {
```

Comment:
Can you swap this out with a MQ variable

Reply:
Done!